### PR TITLE
add Prometheus and must-gather to airflow

### PR DESCRIPTION
**Pull request by Harshith-umesh**

### CODE REVIEW

File: dags/openshift_nightlies/tasks/utils/defaults.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "utils": [
+        {
```
Comment (whitleykeith) [inline dags/openshift_nightlies/tasks/utils/defaults.json:3]: can we remove prometheus as a default and just have must-gather for now?

File: dags/openshift_nightlies/scripts/run_scale-ci-diagnosis.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```
Comment (whitleykeith) [inline dags/openshift_nightlies/scripts/run_scale-ci-diagnosis.sh:1]: can this script be moved into a a dir `dags/openshift_nightlies/scripts/utils/` and rename the file to use either _ or - but not both?

File: dags/openshift_nightlies/tasks/utils/prom_mustgather.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import sys
```
Comment (whitleykeith) [inline dags/openshift_nightlies/tasks/utils/prom_mustgather.py:1]: can this file be renamed to `scale_ci_diagnosis` or something similar to match the getter method at https://github.com/cloud-bulldozer/airflow-kubernetes/pull/22/files#diff-5c327263fbf41890b1c0c7f9db9da88277c03e04638966f33f08f4a4ed14090cR78

**Review by whitleykeith — CHANGES_REQUESTED** [review]

File: dags/openshift_nightlies/scripts/run_scale-ci-diagnosis.sh (hunk 2 above)
Comment (Harshith-umesh) [reply dags/openshift_nightlies/scripts/run_scale-ci-diagnosis.sh]: done

File: dags/openshift_nightlies/tasks/utils/prom_mustgather.py (hunk 3 above)
Comment (Harshith-umesh) [reply dags/openshift_nightlies/tasks/utils/prom_mustgather.py]: done

File: dags/openshift_nightlies/tasks/utils/defaults.json (hunk 1 above)
Comment (Harshith-umesh) [reply dags/openshift_nightlies/tasks/utils/defaults.json:3]: done

**Review by whitleykeith — APPROVED** [review]
lgtm

**Pull request merged**